staff/models.py

```diff
@@ -72,7 +72,7 @@ class Meta:
     ophase = models.ForeignKey(Ophase, models.CASCADE)
     prename = models.CharField(max_length=60, verbose_name=_('first name'))
     name = models.CharField(max_length=75, verbose_name=_('last name'))
-    email = models.EmailField(verbose_name=_("E-Mail-Adresse"))
+    email = models.EmailField(verbose_name=_("E-Mail-Adresse"), unique=True)
```

Comment:
Please do not set unique for the email. There is already a unique_together in line 69. Since cea78800724d172b4bd5d8c4b4835bd2dd9c371d there is a Testcase for it. A email is only unique for a Person in one ophase not globaly. Otherwise a person could not reregister if the old dataset form a previous ophase is not deleted yet.